### ISSUE TITLE
html lang is missing [accessibility issue]

### ISSUE THREAD

**Issue by nyabla** [sycs-climate/climatestrike-wp#3]
According to the [WAVE](https://wave.webaim.org/report#/https://climatestrike.scot/) therre is no tag defining the language. This is important for screen readers and auto translators

**Comment by nyabla** [comment]
I'm gonna add it. It's a quick fix.
